Doc/library/asyncio-eventloop.rst

```diff
@@ -322,7 +322,7 @@ Creating Futures and Tasks
 
 .. method:: loop.create_future()
 
-   Create an :class:`asyncio.Future` object attached to the event loop.
+   Create a :class:`asyncio.Future` object attached to the event loop.
```

Comment:
Wait, this is incorrect! It is read as "an async i o future".

Reply:
I think it is  referring here to `Future` so "a asyncio Future" rather than "an asyncio Event"

Reply:
In ```a :class:`~asyncio.Future` ``` it refers to Future, but in ```an :class:`asyncio.Future` ``` it refers to "async io Future".